packages/commonwealth/client/scripts/views/pages/Community/common/constants.ts

```diff
@@ -35,23 +33,18 @@ export const requirementTypes = [
 ];
 
 // Get eth chain id from the app.config.chains for these chains
-const chainIdsToFind = [
-  'axie-infinity',
-  'cosmos',
-  'ethereum',
-  'injective',
-  'near',
-  'polkadot',
-  'polygon',
-  'solana',
-];
+export const cosmosBaseChainIds = ['cosmos', 'injective'];
+export const ethBaseChainIds = ['axie-infinity', 'ethereum', 'polygon'];
+const chainIdsToFind = [...cosmosBaseChainIds, ...ethBaseChainIds];
 const foundChains = app.config.chains
   .getAll()
   .filter((x) => chainIdsToFind.includes(x.id.toLowerCase()));
-export const chainTypes = chainIdsToFind.map((x, index) => ({
-  value:
-    foundChains.find((y) => y.id.toLowerCase() === x)?.ChainNode?.ethChainId ||
-    index - 999, // TODO: some chains don't have an ethChainId
+export const chainTypes = chainIdsToFind.map((x) => ({
+  chainBase: cosmosBaseChainIds.includes(x) ? 'cosmos' : 'ethereum',
+  value: cosmosBaseChainIds.includes(x)
+    ? 'cosmos' // cosmos and injective have 'cosmos' id
+    : // TODO: polkadot doesn't have an ethChainId
```

Comment:
Is this TODO complete? Makes this line look awkward if it is.

Reply:
oh yes its complete. Removed the comment